crates/router_env/src/logger/types.rs

```diff
@@ -185,6 +185,8 @@ pub enum Flow {
     RoutingUpdateDefaultConfig,
     /// Routing delete config
     RoutingDeleteConfig,
+    /// Payment Method block flow
+    PmBlockFlow,
```

Comment:
Do we need to add separate flows for every sub-routes like unblock, list?
cc: @vspecky